src/PKSim.Presentation/Presenters/Main/StatusBarPresenter.cs

```diff
@@ -43,6 +49,10 @@ public class StatusBarPresenter : IStatusBarPresenter
       private readonly IApplicationConfiguration _applicationConfiguration;
       private int _numberOfReportsBeingCreated;
       public event EventHandler StatusChanged = delegate { };
+      private readonly ConcurrentDictionary<string, bool> _simulations = new ConcurrentDictionary<string, bool>();
+
+      private int activeSimulations => _simulations.Count(x => x.Value);
```

Comment:
rename this variable. I am expecting to get a list of active simulations from the name, not a count
numberOfActiveSimulations or activeSimulationsCount or whatever